src/pages/SettingsPage.js

```diff
@@ -331,6 +169,24 @@ const Settings = ({ toRealmExplorer, currentUserPasswordHash, requestStorageWrit
       <View style={row}>
         <View style={column}>
           <PageInfo columns={pageInfoColumns} />
+          <FlexRow
+            flex={1}
+            justifyContent="center"
+            alignItems="center"
+            style={{ marginTop: 10, maxHeight: 50 }}
```

Comment:
Couple of style questions.. the font looks different for the new setting, and can we have the slider start where the first two settings start?

![settings](https://user-images.githubusercontent.com/9192912/111939766-fc733880-8b31-11eb-9e90-81c29cd293fc.png)